<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnookerAccueil</class>
 <widget class="QWidget" name="SnookerAccueil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1349</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="layoutLogo">
        <item>
         <widget class="QLabel" name="labelLogo">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="ressource.qrc">:/main/pics/logosnooker.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacerLogo">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layoutMenu">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="labelSnooker">
            <property name="font">
             <font>
              <family>Roboto Black</family>
              <pointsize>74</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
  color : #090911;
}</string>
            </property>
            <property name="text">
             <string>SNOOKER</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacerSnooker">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>150</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutBouton">
          <item>
           <layout class="QVBoxLayout" name="layoutBoutonG"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="layoutBoutonM">
            <item>
             <widget class="QGroupBox" name="groupBoxBouton">
              <property name="styleSheet">
               <string notr="true">QGroupBox { 
     border: 1px solid #F1F0CC;
 }</string>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QVBoxLayout" name="layoutNouvellePartie"/>
               </item>
               <item>
                <layout class="QVBoxLayout" name="layoutQuitterPartie"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>250</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="layoutBoutonD"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layoutCredit">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
           </font>
          </property>
          <property name="text">
           <string>Developed by INCEPTUM Junior Entreprise Espr/it. Â®</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
